src/Changelog/ReleaseChangelogAndFetchContents.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\AutomaticReleases\Changelog;
+
+interface ReleaseChangelogAndFetchContents
```

Comment:
The purpose of this interface seems to be unclear and/or overloaded.

I think we should instead:

 1. generate the changelog text
 2. replace it in the pre-existing `CHANGELOG.md` if applicable

Those should be two steps, where one is a pure read operation, and the other is a read+write operation